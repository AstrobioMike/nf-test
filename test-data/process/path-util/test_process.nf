process TEST_PROCESS {

  publishDir "${params.outdir}", mode: 'copy'

  input:
    path input
    path input_json

  output:
     path "*.gz", emit: output_file
     path "output.json", emit: output_json

  """
  gzip -n -c ${input} > ${input}.gz
  cp ${input_json} output.json
  """

}
