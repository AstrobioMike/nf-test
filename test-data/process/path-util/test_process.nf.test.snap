{
    "process_test_1": {
        "content": [
            {
                "output_file": [
                    "input.txt.gz:md5,cad0b2b72339e92e6e53c4d4e6f3a3bc"
                ],
                "output_json": [
                    "output.json:md5,a49d148b49ae6ddae967636d940c1654"
                ]
            }
        ],
        "timestamp": "2022-10-18T11:22:31+0000"
    }
}